app/routines/create_user.rb

```diff
@@ -54,7 +54,12 @@ def guessed_first_name(full_name)
 
   def guessed_last_name(full_name)
     return nil if full_name.blank?
-    full_name.split("\s").drop(1).join(' ')
+    parts = full_name.split("\s")
+    if parts.many?
+      parts.drop(1).join(' ')
+    else
+      full_name
+    end
```

Comment:
This is an attempt to work around identity services that may not provide a last name.  If full_name doesn't contain spaces, then I thought it'd be best to fallback to the first name and still create the record.  Maybe it'd be best to just reject the user though?